zebra-chain/src/sprout/tests/tree.rs

```diff
@@ -0,0 +1,45 @@
+//! Tests for Sprout Note Commitment Trees.
+
+use hex::FromHex;
+
+use crate::sprout::commitment::NoteCommitment;
+use crate::sprout::tests::test_vectors;
+use crate::sprout::tree;
+
+#[test]
+fn empty_roots() {
+    zebra_test::init();
+
+    for i in 0..tree::EMPTY_ROOTS.len() {
+        assert_eq!(
+            hex::encode(tree::EMPTY_ROOTS[i]),
+            test_vectors::HEX_EMPTY_ROOTS[i]
```

Comment:
(See other comment)

If you decide to follow Sapling and invert EMPTY_ROOTS, then you need to check the inverted test vector (see Sapling test)